### PR TITLE
Chore 127 flakey webassembly tests

### LINKED ISSUE #127: Flaky test
The webassembly test is flaky on macOS. It appears to never finish(?) It ends up failing quite frequently!

https://github.com/nearform/node-clinic-flame/blob/f62cb5c8a66cf9eba9becf8a03ee581a176c0454/test/basic.test.js#L112-L139

### PR DESCRIPTION
**Pull request by admataz**
Introduces test reliability in GitHub Actions CI

Trial and error.... I found that specific tests in `test/basic.test.js` fail regularly when tap tests are run in parallel (as they are by default). I think there were some `0x` generated resources being shared across tests. 

Specifying `-j1` in the npm ci test command forces parallelism off globally, but when doing so, the specific test for wasm  regularly failed (times out) - so I've added a specific `t.jobs=2` for the wasm test. 

Tests are now all passing reliably in CI. 

image

Fixes #127

### CODE REVIEW

File: test/basic.test.js [hunk 1]
```diff
@@ -110,6 +110,7 @@ test('cmd - test collect - system info, data files and html', function (t) {
 })
 
 test('cmd - test collect - does not crash on webassembly frames', function (t) {
+  t.jobs = 2
```
Comment (mcollina) [inline test/basic.test.js:113]: I don't think this does what you think it does. There are no substests to run in parallel here.

Comment (admataz) [reply test/basic.test.js:113]: ok thanks. maybe I was just lucky that it passed 4 times in a row.  I thought it was allowing a required parallel process to run within the specific test.

Comment (jasnell) [reply test/basic.test.js]: Well, let's land this and just see. This one has been troublesome for a while

**Comment by mcollina** [comment]
I would recommend the use of a `.taprc` file.

**Pull request merged**